res/values/strings.xml

```diff
@@ -1243,8 +1230,8 @@
 
     <!-- GroupUtil -->
     <plurals name="GroupUtil_joined_the_group">
-        <item quantity="one">%1$s joined the group.</item>
-        <item quantity="other">%1$s joined the group.</item>
+        <item quantity="one">%1$s has been added to the group.</item>
+        <item quantity="other">%1$s has been added to the group.</item>
```

Comment:
An invite immediately also results in an addition. I think "has been added to" makes sense as that is what matters to the user.

Reply:
because joined makes it seem like the person who joined performed the joining action. The fact is that he has been added to the group by someone else, not by himself.